homework-classes/RepoView.js

```diff
@@ -19,8 +19,30 @@
      * @param {Object} repo A repository object.
      */
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
-      console.log('RepoView', repo);
+      this.container.innerHTML = '';
+      const table = createAndAppend('table', this.container);
+      const tr1 = this.addRow(table, 'Repository:', '');
+    createAndAppend('a', tr1.lastChild, {
```

Comment:
Somehow your indentation got screwed up.

Reply:
Thank you so much remarcmij that all valuable feedback.